pyiron_base/project/generic.py

```diff
@@ -522,7 +522,7 @@ def inspect(self, job_specifier):
         """
         return self.load(job_specifier=job_specifier, convert_to_object=False)
 
-    def iter_jobs(self, path=None, recursive=True, convert_to_object=True, status=None):
+    def iter_jobs(self, path=None, recursive=True, convert_to_object=True, status=None, job_type=None):
```

Comment:
Actually, why restrict to `status` and `job_type`? We could in principle extend this for every db column and even pass a masking function to this. But for now, this PR LGTM!